src/libs/decoders/stb_vorbis.h

```diff
@@ -1301,6 +1339,15 @@ static uint8 get8(vorb *z)
       return *z->stream++;
    }
 
+   #ifdef __SDL_SOUND_INTERNAL__
+   {
+   uint8 c;
+   if (SDL_RWread(z->rwops, &c, 1, 1) != 1) { z->eof = TRUE; return 0; }
+   return c;
+   }
+   #endif
```

Comment:
Formatting of this block is a bit inconsistent.

Reply:
Great catch.  Fixed in 003a3fd2.
Yes (also interesting; looks like STB uses indent sizes of three-spaces.)